resip/stack/TransactionMessage.hxx

```diff
@@ -17,7 +18,10 @@ class TransactionMessage : public Message
 
       // indicates this message is associated with a Client Transaction for the
       // purpose of determining which TransactionMap to use
-      virtual bool isClientTransaction() const = 0; 
+      virtual bool isClientTransaction() const = 0;
+
+      // Allows to customise TimerB value independently for each of the client TXs.
+      virtual int getTimerB() const { return Timer::TB; }
```

Comment:
seems strange you have SipMessageOptions setting for timerB, yet you don't return it here.  I'm assuming this should be returning SipMessageOptions timerB value if > 0, or Timer::TB otherwise?

Reply:
oops thought this was still the SipMessage class, disregard.